plugins/module_utils/_api_data.py

```diff
@@ -132,13 +132,23 @@ def join_path(path):
     ),
     ('interface', 'list'): APIData(
         primary_keys=('name', ),
+        fully_understood=True,
         fields={
             'comment': KeyInfo(can_disable=True, remove_value=''),
             'exclude': KeyInfo(),
             'include': KeyInfo(),
             'name': KeyInfo(),
         },
     ),
+    ('interface', 'list', 'member'): APIData(
+        primary_keys=('interface','list', ),
```

Comment:
That should make the sanity tests happy:
```suggestion
        primary_keys=('interface', 'list', ),
```

Reply:
@felixfontein yeah, I just noticed that sanity error and fixed in another commit